bom/pom.xml

```diff
@@ -162,6 +162,11 @@
          </dependency>
          <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-commons-test</artifactId>
```

Comment:
@slaskawi I think it's more because of `TestCacheManagerFactory` and `AbstractInfinispanTest` than because of the listeners :)